frontend/src/utils/utils.ts

```diff
@@ -606,10 +606,17 @@ export const shouldIncludeRepository = (
  * @returns The query string for searching OpenHands repositories
  */
 export const getOpenHandsQuery = (provider: Provider | null): string => {
-  if (provider === "gitlab") {
-    return "openhands-config";
-  }
-  return ".openhands";
+  const providerRepositorySuffix = {
+    gitlab: "openhands-config",
+    azure_devops: "openhands-config",
+    default: ".openhands",
+  } as const;
+
+  return provider && provider in providerRepositorySuffix
+    ? providerRepositorySuffix[
+        provider as keyof typeof providerRepositorySuffix
+      ]
+    : providerRepositorySuffix.default;
```

Comment:
```suggestion
  const providerRepositorySuffix: Record<string, string> = {
    gitlab: "openhands-config",
    azure_devops: "openhands-config",
    default: ".openhands",
  } as const;

  return provider && provider in providerRepositorySuffix
    ? providerRepositorySuffix[provider]
    : providerRepositorySuffix.default;
```

`provider in providerRepositorySuffix` already confirms it's a valid key